src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -688,7 +688,7 @@ def to_numpy(
         # here it's overwritten to actually perform numpy conversion, i.e. return an actual numpy object
         return self.to_pandas().to_numpy(dtype=dtype, na_value=na_value, **kwargs)
 
-    def repartition(self, axis: Any = None) -> "SnowflakeQueryCompiler":
+    def repartition(self, axis: Optional[Any] = None) -> "SnowflakeQueryCompiler":
```

Comment:
why is this change here? if not need by the current pr, please revert the change